### ISSUE TITLE
No Valid Depot Key

### ISSUE THREAD

**Issue by ashyashton** [SteamRE/DepotDownloader#246]
I have tried it with every depot number that the game could possibly be using, and it still doesn't seem to work. It just keeps saying it can't download it because there's "no valid depot key." I'm using the following command, as a guide online told me to do: dotnet DepotDownloader.dll -app 1237620 -depot 1237622 -manifest 7870971855805012604 -username [my username] -password [my password]

I just wanna play the Timberborn Demo until they release the paid version of the game man-

**Comment by xPaw** [comment]
Well, do you own it? Just because it has demo in the name, doesn't always mean you can freely access it.

The demo package is deactivated: https://steamdb.info/sub/429296/history/

**Comment by ashyashton** [comment]
Yes, I do. It's in my library currently, and I've been playing it for a new days now. I just had bad timing and started a few days before they took it down. I posted about it on Reddit, and I saw someone mention that, in order to play the game, you'd basically have to prevent Steam from updating.

Specifically, someone commented on how they think it's odd that there seems to be a "remote kill switch" of sorts, and asked if anyone had found a way to make it run, and someone else responded with the following: _The "remote kill switch" is the steam updater, so you'd have to force steam to not update the game, like playing it offline or something._

This led me to think that maybe I could play the demo some more if I managed to get back to the update from before they took it down, and then I'd set the game to not update until I open the game, then go offline every time I did so to prevent it from updating. I'm not sure if it would actually work, but I've been trying basically all night. I've been using [this guide](https://steamcommunity.com/sharedfiles/filedetails/?id=2353930763) to try and make it work. I got to step 6 and put in the code they gave, and ran into the issue I mentioned above.

TL;DR: Yes, I do own the demo, I'm just trying to go back to a playable version of it.

**Comment by xPaw** [comment]
No Valid Depot Key is telling you that Steam server is not giving you a valid depot key for the demo, which would make sense with the deactivated demo package.

I'm not sure if the actual license also gets deactivated or has special flags set. But there's nothing depot downloader can do here.

**Comment by ashyashton** [comment]
That's unfortunate. Thank you anyways! I'll try to find other options to try and get the demo to work again!
